src/helpers/checkSlug.ts

```diff
@@ -0,0 +1,6 @@
+export function checkSlug(slug: string | undefined): boolean {
+  if (slug !== '' && !slug?.match(/\//)) {
+    return false
+  }
+  return true
+}
```

Comment:
A future PR should move this function to [validate.ts].

[validate.ts]:
https://github.com/codecov/uploader/blob/9f1f09bf2ef2842ea05275b2b0f9eb82b84e78b4/src/helpers/validate.ts

Reply:
No action is required for this annotation.